### PR TITLE
update README.md inputImageFormat method

### PR DESCRIPTION
**Pull request by X-SLAYER**

### CODE REVIEW

File: packages/google_mlkit_commons/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ final InputImageRotation imageRotation =
         InputImageRotation.Rotation_0deg;
```
Comment (fbernaly) [inline packages/google_mlkit_commons/README.md:47]: could you include this fix as well: this lines should be `rotation0deg`

File: packages/google_mlkit_commons/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ final InputImageRotation imageRotation =
...
 
 final InputImageFormat inputImageFormat =
-    InputImageFormatMethods.fromRawValue(cameraImage.format.raw) ??
+    InputImageFormatValue.fromRawValue(cameraImage.format.raw) ??
         InputImageFormat.NV21;
```
Comment (fbernaly) [inline packages/google_mlkit_commons/README.md:51]: could you include this fix as well: this lines should be `nv21`

**Review by X-SLAYER — COMMENTED** [review]
include rotation0deg fix

**Comment by fbernaly** [comment]
@X-SLAYER : also, there is an issue with the analyzer, you need to fix that as well.

**Comment by fbernaly** [comment]
go to 
packages/google_ml_kit/example/lib/nlp_detector_views/entity_extraction_view.dart:135:43

and change it to `annotateText`

**Review by X-SLAYER — COMMENTED** [review]
Update entity_extraction_view.dart

**Comment by fbernaly** [comment]
it is missing L51 in the readme, it should be nv21

**Review by fbernaly — APPROVED** [review]

**Comment by fbernaly** [comment]
thanks @X-SLAYER

**Pull request merged**